benchmark/start-server.ts

```diff
@@ -98,8 +98,7 @@ const yogaMap: Record<string, YogaServerInstance<{}, {}>> = {
     schema,
     logging: false,
     multipart: false,
-    validationCache: false,
-    parserCache: false,
+    parserAndValidationCache: false,
```

Comment:
should this maybe be:
```
cache: boolean | {
    parser: boolean
    validation: boolean
}
```

Reply:
@ardatan what do you think?

Reply:
https://github.com/dotansimha/graphql-yoga/blob/main/packages/graphql-yoga/src/plugins/useParserAndValidationCache.ts#L17
First parser cache caches `DocumentNode` and validation cache keeps the results by `DocumentNode` with `WeakMap`s. So they are not seperated.
If we allow users to have `parser: false` but `validation: true`, we will need to keep the logic here;
https://github.com/dotansimha/graphql-yoga/pull/2777/files#diff-2bb04768b60ebe5776d0a80eafa2b96026b7bae21deda57af2e04bb9e5abb4cbL398

Current options already allow users to disable validation caching or disable both completely.